### PR TITLE
[TIMELINE] Add new attribute to search by measure titles + tests/refactoring

### PR DESCRIPTION
**Pull request by Remg**

### CODE REVIEW

File: src/Entity/Timeline/Measure.php [hunk 1]
```diff
@@ -267,14 +279,13 @@ public function isDeferred(): bool
...
-    public function saveCurrentThemes(): void
-    {
-        $this->savedThemes = clone $this->themes;
-    }
-
     private function getSavedThemes(): Collection
     {
-        return $this->savedThemes ?? new ArrayCollection();
+        if (!$this->savedThemes) {
+            $this->savedThemes = new ArrayCollection();
```
Comment (blazarecki) [inline src/Entity/Timeline/Measure.php]: You don't need to do that if `new ArrayCollection` is in the constructor.

Comment (Remg) [reply src/Entity/Timeline/Measure.php]: Not when it comes from Doctrine.

Comment (blazarecki) [reply src/Entity/Timeline/Measure.php]: Doctrine give you a `PersistedCollection`.

Comment (Remg) [reply src/Entity/Timeline/Measure.php]: Yeah but this is not a collection managed by Doctrine

File: src/Entity/Timeline/Theme.php [hunk 2]
```diff
@@ -120,6 +120,22 @@ public function measureIds(): array
...
     }
 
+    /**
+     * @Algolia\Attribute
+     */
+    public function measureTitles(): array
+    {
+        $titles = [];
+        foreach ($this->measures as $measure) {
+            /* @var $translation MeasureTranslation */
```
Comment (HeahDude) [inline src/Entity/Timeline/Theme.php]: Class first

File: src/Command/ImportTimelineCommand.php [hunk 3]
```diff
@@ -254,7 +254,7 @@ private function parseCSV(string $filepath): array
 
     private function getProfiles(): array
     {
-        /* @var $profile Profile */
+        /* @var Profile $profile */
```
Comment (illusionOfParadise) [inline src/Command/ImportTimelineCommand.php]: `/* @var */` => `/** @var */`

**Pull request merged**